Makefile

```diff
@@ -1,36 +1,28 @@
-.PHONY: build run clean shell all
-SRCS:=Dockerfile requirements.txt requirements-dev.txt app.py
-IMAGE:=quaratine_api
-PORT:=5000:5000
-DARGS:=-it --rm -v $(CURDIR):/app
-DOCKER_RUN?=docker run $(DARGS) -p $(PORT) $(IMAGE)
+.PHONY: all run shell test build clean deps drone-test
 
 all: run
 
-build: .build
-.build: $(SRCS)
-	docker build -t $(IMAGE) .
-	echo > $@
-
 run: build
-	$(DOCKER_RUN)
+	docker-compose up
 
-test:
-	$(DOCKER_RUN) -m flake8
+shell:
+	docker-compose run api bash
 
-shell: build
-	docker run $(DARGS) --entrypoint /bin/bash $(IMAGE)
+build: .build
+.build: requirements.txt requirements-dev.txt Dockerfile
+	docker-compose build
+	echo > $@
+
+clean: 
+	- rm .build
+	- del .build
+	- docker-compose down --rmi all
 
 requirements.txt: requirements.in
-	docker run -it --rm -v $(CURDIR):/usr/src --entrypoint bash python:3.8 -c 'pip install -qqqr /usr/src/requirements.in && pip freeze' > $@
+	docker run -it --rm -v $(CURDIR):/usr/src --entrypoint bash python:3.8 -c "pip install -qqqr /usr/src/requirements.in && pip freeze" > $@
 
 requirements-dev.txt: requirements-dev.in
-	docker run -it --rm -v $(CURDIR):/usr/src --entrypoint bash python:3.8 -c 'pip install -qqqr /usr/src/requirements-dev.in && pip freeze' > $@
-
-deps:
-	pip install -r requirements.txt
-	pip install -r requirements-dev.txt
+	docker run -it --rm -v $(CURDIR):/usr/src --entrypoint bash python:3.8 -c "pip install -qqqr /usr/src/requirements-dev.in && pip freeze" > $@
```

Comment:
I recommend adding the following targets to allow upgrading the requirements.txt files easily:

```
force-clean: clean
       -rm requirements.txt
       -del requirements.txt
       -rm requirements-dev.txt 
       -del requirements-dev.txt 

upgrade: force-clean requirements.txt requirements-dev.txt
```

Add `upgrade` and `force-clean` to the `.PHONY` list.